e2e/routes/apps/todo.test.js

```diff
@@ -27,6 +29,9 @@ test.describe("Todo App", () => {
         title: "test-todo-title",
         description: "test-todo-description",
       };
+      const _res = await apiContext.post(`/api/v1/todos`, {
+        data: todo,
+      });
```

Comment:
why is it duplicated here?

Same post api call is present and captured into `res` below

Reply:
Thanks for noticing I don't knwo how it was added